prac2-pages/Simranpreet Kaur/login_page.dart

```diff
@@ -0,0 +1,181 @@
+// ignore_for_file: prefer_const_literals_to_create_immutables, prefer_const_constructors
+
+import 'package:flutter/material.dart';
+import 'login2.dart' ;
+
+class LoginPage extends StatefulWidget{
+  const LoginPage({Key? key}) : super(key:key);
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+
+}
+class _LoginPageState extends State<LoginPage> {
+  bool check1 = true;
+  @override
+  Widget build(BuildContext context){
+    return Scaffold(
+      backgroundColor: Colors.white,
+      body: SafeArea(
+        child : Center(
```

Comment:
You need to make sure that some widgets align as per the design file. By default, everything is at the center.

- Please try to resolve this as well
If you face any issue, please feel free to ask! :smiley:

Reply:
Resolved! :+1: